packages/realm/src/flags.ts

```diff
@@ -26,4 +26,9 @@ export const flags = {
    * Helps finding places where the app is depending on the now deprecated way of using the package.
    */
   THROW_ON_GLOBAL_REALM: false,
+  /**
+   * When enabled, calling `Realm.clearTestState` will be callable.
```

Comment:
```suggestion
   * Enables calling internal, test-only functions like `Realm.clearTestState`.
```

* better to use active rather than passive voice where possible.
* the effects of this flag are already not limited to that function, so we might as well not lie.

Reply:
I added this.